.github/workflows/build-images-action.yml

```diff
@@ -23,6 +23,18 @@ jobs:
       QUAY_USERNAME: ${{ secrets.QUAY_USERNAME }}
       QUAY_PASSWORD: ${{ secrets.QUAY_PASSWORD }}
       SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK }}
+  build_ironic_cs10:
+    name: Build Ironic container image based on CentOS Stream 10
+    if: github.repository == 'metal3-io/ironic-image'
+    uses: metal3-io/project-infra/.github/workflows/container-image-build.yml@main
+    with:
+      image-name: 'ironic_cs10'
+      pushImage: true
+      env.IMAGE_BUILD_ARGS: 'BASE_IMAGE=quay.io/centos/centos:stream10'
```

Comment:
```suggestion
      env:
        IMAGE_BUILD_ARGS: 'BASE_IMAGE=quay.io/centos/centos:stream10'
```

Reply:
@tuminoid this formatting does not work, I tried it already, it results in an error `Line: 27 Column 5: Failed to match workflow-job: Line: 34 Column 9: Failed to match string: Line: 34 Column 9: Expected a scalar got mapping`

Reply:
OK... maybe that is a `with:` specialty. Lets merge as is then.